src/domain/settings/taxes/selectable-table.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useMemo, useEffect } from "react"
+import clsx from "clsx"
+import { Product, ProductType, ShippingOption } from "@medusajs/medusa"
+import CheckIcon from "../../../components/fundamentals/icons/check-icon"
+import {
+  ColumnInstance,
+  usePagination,
+  useRowSelect,
+  useTable,
+} from "react-table"
+import Table, { TablePagination } from "../../../components/molecules/table"
+import Spinner from "../../../components/atoms/spinner"
+
+type PaginationProps = {
+  limit: number
+  offset: number
+}
```

Comment:
nit: Same as previous comment. Move to `types/shared.ts` and replace both places.